gamemode/modules/chat/cl_chatlisteners.lua

```diff
@@ -78,7 +78,8 @@ local function chatGetRecipients()
 
     receivers = {}
     for _, ply in ipairs(player.GetAll()) do
-        if not IsValid(ply) or ply == LocalPlayer() or ply:GetNoDraw() then continue end
+        hidePly = hook.Run("chatHideRecipient", ply)
```

Comment:
is hidePly global?

Reply:
yes but luckily master Falco caught it:
https://github.com/FPtje/DarkRP/commit/b6237598a9cf27f0e08ce0e3c45af4e9b92ffa10

Reply:
After I mentioned it xD